hw2/03-count.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>03 - Count</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
+    <script defer src="03-count.js"></script>
+  </head>
+  <body class="bg-info">
```

Comment:
Not sure how important this is, but the background color doesn't match the assignment sheet.

Reply:
Yeah I think it's supposed to be bg-info-subtle but it wasn't working for some reason. I figured it's not that big of a deal.